### PR TITLE
Inst UI 3854 instui cli convert

### PR DESCRIPTION
**Pull request by joyenjoyer**
### Test plan
Check if all of the commands in the `instructure-ui/packages/instui-cli/README.md` work as they should.

### CODE REVIEW

**Review by balzss — CHANGES_REQUESTED** [review]
`package-lock.json` is not updated, run `npm install` and add it to this PR

**Comment by joyenjoyer** [comment]
> `package-lock.json` is not updated, run `npm install` and add it to this PR

Done.

File: packages/instui-cli/README.md [hunk 1]
```diff
@@ -38,10 +32,10 @@ instui create component
 
 - **create component** | Create an Instructure UI component. DEPRECATED, will be removed in InstUI v9.
 - **create package** | Create an Instructure UI package. DEPRECATED, will be removed in InstUI v9.
+- **create-from-template** | Copy template source file or directory to a specified destination and replace variables with designated values.
```
Comment (joyenjoyer) [inline packages/instui-cli/README.md:35]: We were having a discussion with Balazs about the `create-from-template` command here. Should this be deprecated as well?
@matyasf

**Review by balzss — APPROVED** [review]

**Review by matyasf — APPROVED** [review]
looks great!

**Pull request merged**